app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -80,12 +80,13 @@ def request_enroll(applicant, is_enhanced_ipp)
     # @param unique_id [String]
     # @param enrollment_code [String]
     # @return [Hash] API response
-    def request_proofing_results(unique_id, enrollment_code)
+    # def request_proofing_results(enrollment)
+    def request_proofing_results(enrollment)
       url = "#{root_url}/ivs-ippaas-api/IPPRest/resources/rest/getProofingResults"
       request_body = {
-        sponsorID: sponsor_id,
-        uniqueID: unique_id,
-        enrollmentCode: enrollment_code,
+        sponsorID: enrollment.sponsor_id,
```

Comment:
I think the sponsor_id has to be an integer

Reply:
True!  Fixed. Thanks.